tests/Sulu/Component/Content/Mapper/ContentMapperTest.php

```diff
@@ -3187,6 +3187,122 @@ public function testContentTypeSwitch()
         $this->mapper->save($testSiteData, 'internal-link', 'sulu_io', 'en', 1, true, $uuid);
     }
 
+    /**
+     * It should delete a node which has children with history.
+     * It should not throw an exception
```

Comment:
I am not sure about this... I've got the feeling that this won't be kept up to date (in case something changes). I would say the method name should already tell what should happen.

`testDeleteWithChildrenHistory` tells me that the that the node should be deleted afterwards, and then we could introduce something like `testInvalidDelete...` to indicate stuff that should fail. I am not even sure that we need the latter one, since a comment with the `@expectedException` annotation should already show that this test should fail. But I would still keep the invalid in the test name, since it might also be possible to set this via `setExcpectedException`.